### PR TITLE
fix(select): multiples accessibility fixes

### PR DESCRIPTION
**Pull request by skhamvon**

### CODE REVIEW

File: packages/stencil/components/select/src/components/osds-select/osds-select.tsx [hunk 1]
```diff
@@ -253,6 +253,13 @@ export class OsdsSelect implements OdsSelect<OdsStencilMethods<OdsSelectMethods>
     }
     this.logger.log('[checkForClickOutside]', arguments, { validity: this.validityState });
     this.controller.closeSurface();
+
+    const selectOptions = this.el.querySelectorAll('osds-select-option');
```
Comment (dpellier) [inline packages/stencil/components/select/src/components/osds-select/osds-select.tsx:257]: can be done in a single loop to increase perf, something like :
```
this.el.querySelectorAll('osds-select-option').forEach((option) => {
  if (this.value && option.getAttribute('value') === this.value) {
    option.setAttribute('selected', 'true');
  } else {
    option.setAttribute('selected', 'false');
  }
})
```

**Review by dpellier — CHANGES_REQUESTED** [review]

File: packages/stencil/components/select/src/components/osds-select/osds-select.tsx (hunk 1 above)
Comment (aesteves60) [reply packages/stencil/components/select/src/components/osds-select/osds-select.tsx:257]: You can also get the list of osds-select-options un the controller with `this.controller.selectOptions`

Comment (skhamvon) [reply packages/stencil/components/select/src/components/osds-select/osds-select.tsx]: Fixed !

**Comment by feoche** [comment]
Changing scope & behavior regarding Select usage

**Pull request closed without merging**